lib/roles_conversion.rb

```diff
@@ -0,0 +1,40 @@
+class RolesConversion
```

Comment:
Well, it's really `NcrRoleCreation`, right? "Create if it doesn't exist." Anyway, not too worried about it.